jrnl/plugins/util.py

```diff
@@ -3,6 +3,18 @@
 # Copyright (C) 2012-2021 jrnl contributors
 # License: https://www.gnu.org/licenses/gpl-3.0.html
 
+from collections import Counter
+
+
+def get_date_counts(journal):
```

Comment:
We've been trying to move everything out of this util file. Since this function is only used in the new plugin, can you move it into there?

Reply:
I figured. On it!

Reply:
Fixed!